### PR TITLE
Chris

### PR DESCRIPTION
**Pull request by chrismeiercs**
User story 7. Needs code review.

### CODE REVIEW

File: src/main/resources/javascript/ViewShipment.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+function getUniqueProducts(){
+
+    var Products = Parse.Object.extend("Products");
+    var query = new Parse.Query(Products);
+    
+    
+
+    query.find({
```
Comment (AdamRison) [inline src/main/resources/javascript/ViewShipment.js]: I would suggest commenting what you're actually doing in this file.  I'm not very familiar with JavaScript, but I'm not code illiterate either.  Reading over this, I'm having a hard time understanding what any of this does.  I get that you access Parse and you run a query, but beyond that, it's unclear.

File: src/main/resources/javascript/ViewShipment.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			
+			$("#products").append("<option>Select Product</option>");
+			uniqueResults.forEach(function(productName){
+				//alert(productName);
+				$("#products").append("<option value='"+productName+"'>"+productName+"</option>");
+			});
+			
+       
+        },
+        error: function(error){
```
Comment (AdamRison) [inline src/main/resources/javascript/ViewShipment.js]: In what case would this error occur?

File: src/main/resources/javascript/ViewShipment.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    var shipmentArray = [];
+
+    productsQuery.find({
+        success: function(results) {
+			$('#shipmentTable tr').has('td').remove();
+            for(var i=0; i < results.length; i++){
+                var object = results[i];
+				
+
```
Comment (AdamRison) [inline src/main/resources/javascript/ViewShipment.js]: Yet again, just a comment or two as to what is happening would be very helpful.  As far as I can tell though, your coding is fine.

File: src/main/resources/templates/viewShipments.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+<script type="text/javascript">
+    //on load
+    $(document).ready(function(){
+
+		//initParse();
+        Parse.initialize("lXpYJrImjyl3YSKDvxX9R6H3GGqIKQrB6WbI6Eu1", "tmfM1mnCsFEE9onzPSAqwKubvgKakqPppNx8dnsq");
+		//getUniqueProducts();
+
+
+		$("#products").change(function(){queryShipments($("#products").val());});
```
Comment (AdamRison) [inline src/main/resources/templates/viewShipments.html]: Is this querying for products from Parse? Throw in a comment.  Also, if it is, you might want to run a test to make sure that it pulls everything that you want if you haven't already.

File: src/main/java/coffeeimport/ViewShipmentController.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+//maps the view shipment page to the shipmentviewer address
+@Controller
+@RequestMapping("/shipmentviewer")
+public class ViewShipmentController {
+
+    @RequestMapping(method = RequestMethod.GET)
+    public String printWelcome(Model model) {
+
+        return "ViewShipments";
+    }
```
Comment (AdamRison) [inline src/main/java/coffeeimport/ViewShipmentController.java]: Very concise method.. Nice.

File: src/main/resources/javascript/ViewShipment.js (hunk 1 above)
Comment (chrismeiercs) [inline src/main/resources/javascript/ViewShipment.js]: This file wasn't meant to be committed. Resubmitting.

**Pull request closed without merging**